README.md

```diff
@@ -15,16 +15,23 @@ This guide will walk you through the steps to configure and run the Azure Key Va
 ### Install the Secrets Store CSI Driver 
 **Prerequisites**
 
-Recommended Kubernetes version: v1.16.0+
+Recommended Kubernetes version: 
+- For linux - v1.16.0+
+- For windows - v1.18.0+
 
-💡 Make sure you have followed the [Installation guide for the Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver#usage).
+💡 Make sure you have followed the [Installation guide for the Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver#usage) to install the driver.
 
 ### Install the Azure Key Vault Provider
 
 ```bash
 kubectl apply -f https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/deployment/provider-azure-installer.yaml
 ```
 
+For windows nodes
```

Comment:
For now added the windows installation in the same readme. Wondering if we need to just create a separate readme for windows installation?

Reply:
I think it's ok for now.